app/assets/scripts/views/locations-hub/filter.js

```diff
@@ -48,7 +48,7 @@ export default function Filter({ countries, parameters, sources }) {
   sortList(countries);
   sortList(parameters);
 
-  const parameterNames = [...new Set(parameters.map(p => p.name))];
+  //const parameterNames = [...new Set(parameters.map(p => p.name))];
```

Comment:
the [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) ensures we have unique parameters in the filter dropdown

Reply:
@AliceR I realized that the parameter list contains params with duplicate name but different id and other info (e.x. CO mass / CO Conc both have name === co). I will add a filter by id

Reply:
i think actually that we do not have duplicates (it seemed that way earlier because we were only using name which is non-unique), @bitner can you please confirm?

Reply:
parameter could still have duplicates, right now parameterId is the only thing guaranteed to be unique (or the combination of parameter and unit) -- that being said, displayName I believe is now unique and I could enforce that